trpcultivate_phenotypes/tests/src/Kernel/TripalImporter/TraitImporterProcessValidationTest.php

```diff
@@ -1477,4 +1477,575 @@ public function testProcessDuplicateTraitsFailures(array $failures, array $expec
     }
   }
 
+  /**
+   * Data Provider for triggering exceptions in checkValidationStatusArray().
+   *
+   * @return array
+   *   Each scenario is an array with the following:
+   *   - The validation status array returned by a single validator and where
+   *     applicable, a single row of the input file (ie. a row-level validator).
```

Comment:
I think this has become outdated. Perhaps it should be

```suggestion
   *   - The validation status array returned by a single validator.
```

Reply:
Good catch! Addressed in commit [41d238b](https://github.com/TripalCultivate/TripalCultivate-Phenotypes/pull/139/commits/41d238bfd13b84e9ee9aa2586074356d3e886f2b)